samples/SampleDialogApp/Views/MainWindow.axaml.cs

```diff
@@ -15,9 +15,9 @@ public MainWindow()
         }
 
         // When referencing Avalonia package, XamlNameReferenceGenerator
-        ////private void InitializeComponent()
-        ////{
-        ////    AvaloniaXamlLoader.Load(this);
-        ////}
+        private void InitializeComponent()
```

Comment:
If there is an issue with InitializeComponent in preview5, then XamlNameReferenceGenerator needs to be upgraded as well.

Reply:
Good look.
Also in the SampleMvvmApp project, I just realized it's using the old `.xaml` file naming and not `.axaml` files which automatically get picked up by the `XamlNameReferenceGenerator`